### PR TITLE
Feature/sticky confirm buttons [PIL-547]

### PR DESCRIPTION
**Pull request by vyvil**
This PR adds reusable ReviewAndConfirm layout
and updates Exchange review screen to make confirmation button floating.

### CODE REVIEW

File: src/components/ReviewAndConfirm/ReviewAndConfirm.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+const WarningMessage = styled(Paragraph)`
+  text-align: center;
+  color: ${themedColors.negative};
+  padding-bottom: ${spacing.small}px;
+`;
+
+const ValueHolder = ({ value, valueArray, isLoading }) => {
+  if (isLoading) return <Spinner width={20} height={20} />;
+  if (valueArray && valueArray.length) return valueArray.map((val) => (<Value key={val}>{val}</Value>));
```
Comment (jsidorenko) [inline src/components/ReviewAndConfirm/ReviewAndConfirm.js:78]:
```suggestion
  if (!isEmpty(valueArray)) return valueArray.map(val => <Value key={val}>{val}</Value>);
```

**Comment by vyvil** [comment]
Can you specify Android device, @graszka22?

**Comment by graszka22** [comment]
@vyvil https://www.gsmarena.com/sony_xperia_xa2-8986.php with android 9

**Comment by graszka22** [comment]
it's kinda like it's scrolling to the top itself?

**Comment by vyvil** [comment]
@graszka22,
Fixed TextInput focus on iOS.
Could not replicate this issue on Android, that you've reported;/.
I've attempted to fix it by disabling this Keyboard avoidance scroll wrapper adjustments on Android (as we do have windowSoftInputMode set to adjustResize on Android). Could you, please, take a look if this fixes screen jumping on your side? :)

**Comment by graszka22** [comment]
Nah, it doesn't fix the Android side. I've made a fix myself, since I can reproduce it: https://github.com/pillarwallet/pillarwallet/pull/2015. I think the problem was that `ContainerWithHeader` also has a `ScrollWrapper`. You can merge it to your branch if you want ;)

**Pull request merged**